src/service/Service.hpp

```diff
@@ -72,12 +84,14 @@ class Service {
      */
     static void finalize_globals();
 
+    /**
+     * @brief Provides the number of references to the Service object
+     *
+     * @return uint32_t Number of references to the Service object
+     */
+    static uint32_t globals_reference_count();
```

Comment:
Why did we make this method public? Where is it used?

Reply:
(this comment applies also to Recording Service, I guess)

Reply:
You're right, we don't really need it. I've removed it from the Service class.